src/components/HelFormFields/utils/SelectHelper.js

```diff
@@ -70,9 +70,9 @@ export const ariaOverrides = (intl, placeholderId, resource) => {
  * @param {Object[]} value
  * @returns {{id: (string)}}
  */
-export const optionsMessageValue = (value = []) => {
+export const optionsMessageValue = (value = [], messageId = '') => {
     return {
-        id: value.length > 2 ? 'search-no-results' : 'search-minimum-length',
+        id: value.length > 2 ? 'search-no-results' : (messageId ? messageId : 'search-minimum-length'),
```

Comment:
If the min search chars is 4, shouldn't this "no search results" also be at 4 for organization search? This function is used by other components which might use min 3.

Reply:
Fixed! Except the organization, other searches user min char 3.